### PR TITLE
desktop: Change 'socket mode' text to 'TCP Connections'

### PR DESCRIPTION
**Pull request by Dinnerbone**

### CODE REVIEW

**Comment by torokati44** [comment]
Are the string IDs good as is?

**Comment by Dinnerbone** [comment]
Unless we deliberately want to invalidate every current translation. Do we?

**Comment by torokati44** [comment]
Since the original language text (English) changes, they are pretty much already invalid, just not technically, no?

**Comment by sleepycatcoding** [comment]
Shouldn't we change the command line flag too, no?

File: desktop/src/gui/open_dialog.rs [hunk 1]
```diff
@@ -155,26 +155,26 @@ impl OpenDialog {
...
+                    .selected_text(match self.options.tcp_connections {
+                        SocketMode::Unrestricted => text(&self.locale, "tcp-connections-allow"),
+                        SocketMode::Ask => text(&self.locale, "tcp-connections-ask"),
+                        SocketMode::Deny => text(&self.locale, "tcp-connections-deny"),
                     })
                     .show_ui(ui, |ui| {
                         ui.selectable_value(
-                            &mut self.options.socket_mode,
+                            &mut self.options.tcp_connections,
                             SocketMode::Unrestricted,
```
Comment (sleepycatcoding) [inline desktop/src/gui/open_dialog.rs:168]: SocketMode::Unrestricted should be renamed to SocketMode::Allow, if we want full parity with command line flag and desktop GUI.

Comment (Dinnerbone) [reply desktop/src/gui/open_dialog.rs:168]: Good catch!

File: desktop/src/gui/open_dialog.rs [hunk 2]
```diff
@@ -155,26 +155,26 @@ impl OpenDialog {
...
+                        SocketMode::Ask => text(&self.locale, "tcp-connections-ask"),
+                        SocketMode::Deny => text(&self.locale, "tcp-connections-deny"),
                     })
                     .show_ui(ui, |ui| {
                         ui.selectable_value(
-                            &mut self.options.socket_mode,
-                            SocketMode::Unrestricted,
+                            &mut self.options.tcp_connections,
+                            SocketMode::Allow,
                             text(&self.locale, "socket-mode-unrestricted"),
```
Comment (sleepycatcoding) [inline desktop/src/gui/open_dialog.rs:169]: Shouldn't this be tcp-connections-allow?

File: desktop/src/gui/open_dialog.rs [hunk 3]
```diff
@@ -155,26 +155,26 @@ impl OpenDialog {
...
                         );
                         ui.selectable_value(
-                            &mut self.options.socket_mode,
+                            &mut self.options.tcp_connections,
                             SocketMode::Ask,
                             text(&self.locale, "socket-mode-ask"),
```
Comment (sleepycatcoding) [inline desktop/src/gui/open_dialog.rs:174]: Here too.

File: desktop/src/gui/open_dialog.rs [hunk 4]
```diff
@@ -155,26 +155,26 @@ impl OpenDialog {
...
                         );
                         ui.selectable_value(
-                            &mut self.options.socket_mode,
+                            &mut self.options.tcp_connections,
                             SocketMode::Deny,
                             text(&self.locale, "socket-mode-deny"),
```
Comment (sleepycatcoding) [inline desktop/src/gui/open_dialog.rs:179]: This too.

Comment (Dinnerbone) [reply desktop/src/gui/open_dialog.rs]: Fixed all three, thank you!

**Review by sleepycatcoding — APPROVED** [review]

**Pull request merged**